src/stylesheets/modal.css

```diff
@@ -10,7 +10,14 @@ please add them to the list:
 - styleSheetUtils.js
 - product-landing-page-tests.js
 */
-
+#fcc_test_suite_wrapper {
```

Comment:
The iframe needs a wrapper to be present in the document on load, and since the `<div id="mocha">` styles are injected automatically via Webpack, I'm hitching the test suite wrapper style into that stylesheet as a single exception. I tried injecting the test suite iframe css via Webpack without success, so for that I am using jQuery to insert a `<style>` element with Webpack-loaded css string separately. I am adding comments to the index.js and will update the PR momentarily.